scripts/dev.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+npx nodemon --watch server.js server.js
```

Comment:
I moved all those to bash files because I thought this one would be more complicated but… 😄 Worth it anyway imo.